### ISSUE TITLE
Android testing is failing with Bazel 2.0.0

### ISSUE THREAD

**Issue by meteorcloudy** [android/testing-samples#303]
https://buildkite.com/bazel/android-testing/builds/611
```
(00:01:58) ERROR: /workdir/ui/espresso/AccessibilitySample/BUILD.bazel:28:1: Couldn't build file ui/espresso/AccessibilitySample/libBasicSampleTestLib.jar: Building ui/espresso/AccessibilitySample/libBasicSampleTestLib.jar (1 source file) failed (Exit 1)
ui/espresso/AccessibilitySample/app/src/androidTest/java/com/example/android/testing/espresso/AccessibilitySample/AccessibilityChecksTest.java:24: error: package androidx.test.espresso.accessibility does not exist
import androidx.test.espresso.accessibility.AccessibilityChecks;
                                           ^
ui/espresso/AccessibilitySample/app/src/androidTest/java/com/example/android/testing/espresso/AccessibilitySample/AccessibilityChecksTest.java:40: error: cannot find symbol
        AccessibilityChecks.enable();
        ^
  symbol:   variable AccessibilityChecks
  location: class AccessibilityChecksTest
ui/espresso/AccessibilitySample/app/src/androidTest/java/com/example/android/testing/espresso/AccessibilitySample/AccessibilityChecksTest.java:42: error: package R does not exist
            onView(withId(R.id.openBrowserButton)).perform(click());
```

**Comment by meteorcloudy** [comment]
This is failing since https://buildkite.com/bazel/android-testing/builds/573
The culprit is https://github.com/android/testing-samples/commit/f3f00fcbdb551bb258e3bf954dd004e5aafc8b6b

@jin, should we rollback this to fix the build?

**Comment by jin** [comment]
cc @brettchabot 

```

(00:24:57) ERROR: /workdir/ui/espresso/AccessibilitySample/BUILD.bazel:6:1: Couldn't build file ui/espresso/AccessibilitySample/libBasicSampleLib.jar: Building ui/espresso/AccessibilitySample/libBasicSampleLib.jar (1 source file) failed (Exit 1)
--
  | ui/espresso/AccessibilitySample/app/src/main/java/com/example/android/testing/espresso/AccessibilitySample/MainActivity.java:37: error: package R does not exist
  | setContentView(R.layout.activity_main);
  | ^
  | (00:24:58) INFO: From Linking static android resource library for //ui/espresso/AccessibilitySample:BasicSampleLib:
  | Dec 04, 2019 12:24:57 AM com.google.devtools.build.android.AndroidManifest parseFrom
  | WARNING:
  | CONFIGURATION: bazel-out/android-armeabi-v7a-fastbuild/bin/ui/espresso/AccessibilitySample/BasicSampleLib_processed_manifest/AndroidManifest.xml has no minSdkVersion. Using 1.
  | (00:24:58) ERROR: /workdir/ui/espresso/AccessibilitySample/BUILD.bazel:28:1: Couldn't build file ui/espresso/AccessibilitySample/libBasicSampleTestLib.jar: Building ui/espresso/AccessibilitySample/libBasicSampleTestLib.jar (1 source file) failed (Exit 1)
  | ui/espresso/AccessibilitySample/app/src/androidTest/java/com/example/android/testing/espresso/AccessibilitySample/AccessibilityChecksTest.java:24: error: package androidx.test.espresso.accessibility does not exist
  | import androidx.test.espresso.accessibility.AccessibilityChecks;
  | ^
  | ui/espresso/AccessibilitySample/app/src/androidTest/java/com/example/android/testing/espresso/AccessibilitySample/AccessibilityChecksTest.java:40: error: cannot find symbol
  | AccessibilityChecks.enable();
  | ^
  | symbol:   variable AccessibilityChecks
  | location: class AccessibilityChecksTest
  | ui/espresso/AccessibilitySample/app/src/androidTest/java/com/example/android/testing/espresso/AccessibilitySample/AccessibilityChecksTest.java:42: error: package R does not exist
  | onView(withId(R.id.openBrowserButton)).perform(click());
  | ^
  | (00:25:00) ERROR: /workdir/ui/espresso/AccessibilitySample/BUILD.bazel:6:1: Couldn't build file ui/espresso/AccessibilitySample/libBasicSampleLib.jar: Building ui/espresso/AccessibilitySample/libBasicSampleLib.jar (1 source file) failed (Exit 1)
  | ui/espresso/AccessibilitySample/app/src/main/java/com/example/android/testing/espresso/AccessibilitySample/MainActivity.java:37: error: package R does not exist
  | setContentView(R.layout.activity_main);
  | ^
  | (00:25:00) ERROR: /workdir/ui/espresso/AccessibilitySample/BUILD.bazel:28:1: Couldn't build file ui/espresso/AccessibilitySample/libBasicSampleTestLib.jar: Building ui/espresso/AccessibilitySample/libBasicSampleTestLib.jar (1 source file) failed (Exit 1)
  | ui/espresso/AccessibilitySample/app/src/androidTest/java/com/example/android/testing/espresso/AccessibilitySample/AccessibilityChecksTest.java:24: error: package androidx.test.espresso.accessibility does not exist
  | import androidx.test.espresso.accessibility.AccessibilityChecks;
  | ^
  | ui/espresso/AccessibilitySample/app/src/androidTest/java/com/example/android/testing/espresso/AccessibilitySample/AccessibilityChecksTest.java:40: error: cannot find symbol
  | AccessibilityChecks.enable();
  | ^
  | symbol:   variable AccessibilityChecks
  | location: class AccessibilityChecksTest
  | ui/espresso/AccessibilitySample/app/src/androidTest/java/com/example/android/testing/espresso/AccessibilitySample/AccessibilityChecksTest.java:42: error: package R does not exist
  | onView(withId(R.id.openBrowserButton)).perform(click());


```

It looks like the `BasicSampleLib` target is missing a dependency on `androidx.test.espresso.accessibility`?

```
android_library(
    name = "BasicSampleLib",
    srcs = glob(["app/src/main/**/*.java"]),
    custom_package = "com.example.android.testing.espresso.BasicSample",
    manifest = "app/src/main/AndroidManifest.xml",
    resource_files = glob(["app/src/main/res/**/*"]),
    deps = [
        artifact("com.google.guava:guava")
    ],
)
```

**Comment by jin** [comment]
https://github.com/android/testing-samples/pull/309 fixes the build, but disables it from CI because of https://github.com/android/android-test/issues/376
